tests/components/brunt/test_config_flow.py

```diff
@@ -0,0 +1,157 @@
+"""Test the Brunt config flow."""
+from unittest.mock import patch
+
+import pytest
+
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.components.brunt.const import DOMAIN
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+
+from tests.common import MockConfigEntry
+
+CONF = {CONF_USERNAME: "test-username", CONF_PASSWORD: "test-password"}
+
+
+async def _flow_submit(hass, context={"source": config_entries.SOURCE_USER}, data=CONF):
```

Comment:
Please don't put a mutable object as default argument.

Reply:
I'd remove this helper function and just inline the call where we need it. It's not that much shorter.